packages/mdc-slider/_variables.scss

```diff
@@ -16,3 +16,6 @@
 
 $mdc-slider-track-opacity: .38;
 $mdc-slider-thumb-handle-active-size: 1.66667;
+$mdc-slider-disabled-active-rail-opacity: .32;
```

Comment:
I'd rename this variable to be closer to the element it modifies. Something like `$mdc-slider-disabled-track-opacity`.

Reply:
Sounds good!